src/main/kotlin/model/Referee.kt

```diff
@@ -0,0 +1,18 @@
+package model
+
+class Referee(private val racingCars: RacingCars) {
```

Comment:
```suggestion
class Referee(private val racingCars: List<RacingCar>) {
```
What’s the difference, and what benefits does this approach offer?

Reply:
The main benefit of First-class collection is "the detail of implementation for RacingCars can be hiden".
As you mentioned, the `moveAll()` can be positioned into RacingCars, it can make easier to read RacingCarGame.
Also, If I want to change data structure for storing RacingCars, (like List -> Map) I can change it easily.